lib/compilers/dotnet.ts

```diff
@@ -103,15 +103,21 @@ class DotNetCompiler extends BaseCompiler {
         ];
     }
 
+    getCompilerOptions() {
+        return this.lang.id === 'il'
+            ? ['-nologo', '-quiet', '-optimize']
+            : ['build', '-c', this.buildConfig, '-v', 'q', '--nologo', '--no-restore', '/clp:NoSummary'];
+    }
+
     get configurableSwitches() {
         return [
-            '-O',
+            '-o',
```

Comment:
I'm going to have to trust you know what you're doing here :D

Reply:
I changed the later check to `this.configurableOptions.includes(currentOption.toLowerCase())`, especially, added the `.toLowerCase()`, as some options are case-insensitive while some are not. So the configurable options/switches here should be all lower cases. 
btw I think maybe we can allow any options/switches without filtering them.

Reply:
Righto; thanks